src/surrogates/inc/InterpolationSurrogateBuilder.h

```diff
@@ -47,7 +47,7 @@ namespace QUESO
     //! Constructor
     /*! We do not take a const& to the data because we want to compute and
         set the values directly. */
-    InterpolationSurrogateBuilder( InterpolationSurrogateData<V,M>& data );
+    InterpolationSurrogateBuilder( InterpolationSurrogateDataSet<V,M>& data );
```

Comment:
My thinking was that the single version would be the exception rather than the rule and that it's more code to maintain. But, if you would rather both be kept around, I'll go back and add.